hawc/apps/assessment/models.py

```diff
@@ -569,6 +595,9 @@ class BaseEndpoint(models.Model):
     # in assessment; major use case in HAWC.
 
     name = models.CharField(max_length=128, verbose_name="Endpoint/Adverse outcome")
+    name_term = models.ForeignKey(
```

Comment:
This also shows up on the epi `Outcome` form and likely the invitro `IVEndpoint` form, since they both subclass this `BaseEndpoint`. Maybe remove from these forms until/if we add this functionality to those endpoints.

![image](https://user-images.githubusercontent.com/46504563/93641020-97dff100-f9c9-11ea-83aa-f73bbe9ed4c2.png)

Reply:
ohhh, great find! I'll fix this!